src/Symfony/Component/DependencyInjection/Compiler/MergeExtensionConfigurationPass.php

```diff
@@ -31,7 +31,7 @@ public function process(ContainerBuilder $container)
         $aliases = $container->getAliases();
         $exprLangProviders = $container->getExpressionLanguageProviders();
 
-        foreach ($container->getExtensions() as $extension) {
```

Comment:
So, is this in the order of registration? If so, isn't the bug more inside the merge function of the ContainerBuilder than in the order of the prepends?